tests/end-to-end/upgrade/pkg/tests/eventmesh-migration/migrate-eventmesh/eventmesh.go

```diff
@@ -0,0 +1,96 @@
+package migrateeventmesh
+
+import (
+	appconnector "github.com/kyma-project/kyma/components/application-operator/pkg/client/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+	serving "knative.dev/serving/pkg/client/clientset/versioned"
+
+	sc "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+	appBroker "github.com/kyma-project/kyma/components/application-broker/pkg/client/clientset/versioned"
```

Comment:
This import should be more specific, like "...clientset"

Reply:
done